tools/bin/ldd_fix

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
```

Comment:
I'd maybe add a comment to this file explaining why it's needed at all, just for future reference.